server.py

```diff
@@ -0,0 +1,92 @@
+from bottle import route, run, template
+import requests
+import datetime
+from datetime import datetime, timedelta
+
+def get_date(date):
+    #2021-08-26
+    endpoint = f'https://www.recreation.gov/api/timedentry/availability/facility/10086745?date={date}'
```

Comment:
Where did you find info about their API? Didn't realize they had one?

Reply:
Right click chrome -> inspect -> go to networks tab

![I](https://i.imgur.com/jpISWeS.png)

Then press one of the calendar dates, you'll see that some new requests show up at the bottom of the page

![2](https://i.imgur.com/oY9spxQ.png)

Now that shows the http request and response being sent. You can even directly access the page (since it's a get request) and it'll return raw json

https://www.recreation.gov/api/timedentry/availability/facility/10086745?date=2021-08-26

![aa](https://i.imgur.com/suUE32Q.png)

Reply:
Ah, so it may not be public, but this is a cool hack. Thanks!